src/OT/Layout/GPOS/AnchorFormat3.hh

```diff
@@ -91,11 +91,18 @@ struct AnchorFormat3
       }
     }
 
-    /* in case that all axes are pinned or no variations after instantiation,
+
+    /* downgrade to format1 if:
+     * 1. both Device offsets are NULL
+     * 2. Device is VariationDevice and all axes are pinned or no variations after instantiation,
```

Comment:
Now the comment is stale. Just remove it is fine.

Reply:
Done.